### PR TITLE
[rhobs-logs] Upgrade Loki to 2.7.2

### PR DESCRIPTION
**Pull request by aminesnow**
Upgrade Loki from 2.6.1 to 2.7.2.

**Depends on:** [observatorium/observatorium/pull/506](https://github.com/observatorium/observatorium/pull/506)
**Ticket:** [LOG-3052](https://issues.redhat.com/browse/LOG-3052)

### CODE REVIEW

**Comment by aminesnow** [comment]
CC @periklis

**Review by periklis — CHANGES_REQUESTED** [review]
Overall looks good to me. Maybe also set the `LOKI_IMAGE_TAG` in `observatorium-logs-template.jsonnet` to `v2.7.2` as to be informative enough for dev-testing that we use 2.7

File: services/observatorium-logs-template.jsonnet [hunk 1]
```diff
@@ -26,7 +26,7 @@ local obs = import 'observatorium.libsonnet';
     { name: 'ALERTMANAGER_EXTERNAL_URL', value: 'https://observatorium-alertmanager-mst.api.stage.openshift.com' },
     { name: 'RULES_OBJSTORE_S3_SECRET', value: 'rules-objstore-stage-s3' },
     { name: 'STORAGE_CLASS', value: 'gp2' },
-    { name: 'LOKI_IMAGE_TAG', value: '2.6.1' },
+    { name: 'LOKI_IMAGE_TAG', value: '2.7.2' },
```
Comment (aminesnow) [inline services/observatorium-logs-template.jsonnet:29]: @periklis this has been already set

Comment (periklis) [reply services/observatorium-logs-template.jsonnet:29]: I know I know, but adding this here too improves visibility when you use this repo only, e.g. during creating/testing a PR here without considering observatorium/observatorium

Comment (aminesnow) [reply services/observatorium-logs-template.jsonnet:29]: I'm sorry I've no idea what you're talking about here..

**Review by periklis — APPROVED** [review]

File: services/observatorium-logs-template.jsonnet (hunk 1 above)
Comment (aminesnow) [reply services/observatorium-logs-template.jsonnet:29]: Can you please suggest a change so that I understand better

**Pull request merged**